docs/api.md

```diff
@@ -274,6 +275,13 @@ await context.close();
 - [browserContext.setPermissions(origin, permissions[])](#browsercontextsetpermissionsorigin-permissions)
 <!-- GEN:stop -->
 
+#### event: 'close'
+
+Emitted when Browser context gets closed. This might happen because of one of the following:
+- Browser context is closed.
```

Comment:
Do we emit page close on disconnect? Maybe we should? I think we should be consistent.

Reply:
Should, we can do that.

Reply:
@JoelEinbinder turns out it is fired. Added a test.